[2025-03-14T21:10:01.591+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T21:10:01.601+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_weather_etl.extract scheduled__2025-03-14T21:05:00+00:00 [queued]>
[2025-03-14T21:10:01.786+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_weather_etl.extract scheduled__2025-03-14T21:05:00+00:00 [queued]>
[2025-03-14T21:10:01.786+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-14T21:10:01.793+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): extract> on 2025-03-14 21:05:00+00:00
[2025-03-14T21:10:01.802+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1596) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-14T21:10:01.804+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'open_weather_etl', 'extract', 'scheduled__2025-03-14T21:05:00+00:00', '--job-id', '203', '--raw', '--subdir', 'DAGS_FOLDER/open_weather_dag.py', '--cfg-path', '/tmp/tmp9dw8mjn4']
[2025-03-14T21:10:01.805+0000] {standard_task_runner.py:72} INFO - Started process 1597 to run task
[2025-03-14T21:10:01.806+0000] {standard_task_runner.py:105} INFO - Job 203: Subtask extract
[2025-03-14T21:10:01.862+0000] {task_command.py:467} INFO - Running <TaskInstance: open_weather_etl.extract scheduled__2025-03-14T21:05:00+00:00 [running]> on host c20e6545b232
[2025-03-14T21:10:01.905+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='lherrera5034@gmail.com' AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='open_weather_etl' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T21:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-14T21:05:00+00:00'
[2025-03-14T21:10:01.906+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T21:10:01.931+0000] {logging_mixin.py:190} INFO - host
[2025-03-14T21:10:01.932+0000] {logging_mixin.py:190} INFO - 5432
[2025-03-14T21:10:01.932+0000] {logging_mixin.py:190} INFO - your_username
[2025-03-14T21:10:01.932+0000] {logging_mixin.py:190} INFO - your_password
[2025-03-14T21:10:01.932+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-14T21:10:01.940+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T21:10:01.940+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=open_weather_etl, task_id=extract, run_id=scheduled__2025-03-14T21:05:00+00:00, execution_date=20250314T210500, start_date=20250314T211001, end_date=20250314T211001
[2025-03-14T21:10:01.992+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T21:10:02.006+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-14T21:10:02.007+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
